apps/nextjs/src/app/layout.tsx

```diff
@@ -93,6 +93,8 @@ export default function RootLayout({ children }: Readonly<RootLayoutProps>) {
                   <CookieConsentProvider>
                     <AnalyticsProvider
                       avoOptions={{
+                        webDebugger: false,
+                        inspector: undefined,
```

Comment:
@tomwisecodes is this needed?

Reply:
This was me trying to get rid of the annoying avo thing. I can put it in a sepearate PR